README.md

```diff
@@ -132,12 +124,6 @@ Check [examples](examples) and [interop](interop)(Interoperability Test) for som
 
 2. [Benchmark](benchmark) followed by official spec
 
-## Sponsors
```

Comment:
I'll have to defer to @tony612 on whether it's ok to remove this. I assume it is.